doc(csi metrics): add csi metrics support doc (#989)
longhorn/website: content/docs/1.8.0/monitoring/metrics.md

| longhorn_backup_backing_image_actual_size_bytes | Actual size of this backup backing image | longhorn_backup_backing_image_actual_size_bytes{backup_backing_image="parrot"} 3.3554432e+07 |
| longhorn_backup_backing_image_state | State of this backup backing image: 0=New, 1=Pending, 2=InProgress, 3=Completed, 4=Error, 5=Unknown | longhorn_backup_backing_image_state{backup_backing_image="parrot"} 3 |

## CSI

> CSI sidecar component has built-in metrics for users to get insights of CSI operations including total count, error count and call latency. Longhorn exposes the service for each csi sidecar to allow users to access these metrics.

Reviewer: ```suggestion
The CSI sidecar component has built-in metrics for users to get insights into CSI operations. The CSI operations metrics cover total count, error count, and call latency. Longhorn exposes the service for each CSI sidecar component to allow users to access these metrics.
```

Author: fixed. thanks

Author: we have changed a bit of the content, PTAL, thanks

| Name | Port |
|---|---|
| longhorn-csi-attacher | 8000 | 
| longhorn-csi-provisioner | 8000 |
| longhorn-csi-resizer | 8000 |
| longhorn-csi-snapshotter | 8000 |

> The metrics provided by the CSi sidecar are in histogram format. For example, you can obtain metrics to observe the time it takes to create a Longhorn Volume for the PVC.

Reviewer: ```suggestion
The metrics provided by the CSI sidecar component are provided in a histogram format. For example, you can obtain metrics observing the time it takes to create a Longhorn Volume for the PVC.
```

Author: fixed
